### ISSUE TITLE
false positive 'unterminated string literal' for multi-line style bindings

### ISSUE THREAD

**Issue by janosh** [pheuter/svelte-check-rs#77]
`svelte-check-rs` reports errors for valid svelte 5 multi-line style bindings that `svelte-check` accepts.

## minimal reproduction

```svelte
<span
  style:transform="rotate({angle}deg)
  translate({dx}px, {dy}px)"
>
</span>
```

## errors

```
Error: Unterminated string literal. (ts(TS1002))
Error: ',' expected. (ts(TS1005))
```

## verification

```bash
npx svelte-check-rs --threshold error  # fails
npx svelte-check --threshold error     # passes
```

- svelte-check-rs: 0.6.9
- svelte: 5.x
